src/DynamoManipulation/PointOnCurveManipulator.cs

```diff
@@ -83,30 +83,56 @@ protected override void AssignInputNodes()
                 if (null == curve)
                     return;
             }
-            catch (Exception) 
-            { 
-                return; 
+            catch
+            {
             }
         }
 
+        // TODO: Implement this method 
```

Comment:
you mean implement this method without relying on LibG?

Reply:
I am wondering if it makes more sense to implement these methods in LibG or in the Primitives lib so they can be tested against the same curve tests we run on LibG geo? (rather than here?)

Reply:
Yes, it's about implementing it without relying on LibG. I was thinking to create a new geometry primitives project and move these functions into that but it could make more sense to make it part of LibG.